src/views/frontPage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const frontPage = (props) => (
```

Comment:
When naming your components, could make sure they start with a capital letter i.e `FrontPage`

Reply:
@the22mastermind I would prefer that rule for the class components, others we use camelcase naming

Reply:
I'm not sure that would be a best practice.
Actually I would like to suggest the following:
1. We all use function-based components and hooks.
2. Other functions like those in Utils, we use camelCase naming.

Check out this article for more info: https://blog.pusher.com/troubleshoot-react-errors/

Reply:
@the22mastermind Although I am not used to working with states in non-class based components, I accept your suggestion, I am willing to learn as much as possible from you guys. Thanks!

Reply:
I used small letter because it's functional based component

Reply:
I made sure to import it as FrontPage

Reply:
Yes, the import name, filename and component name need to be the same. Please check out the link I shared above.

Reply:
Yeah, I agree with you but the issue arises when you try to use the component with small capital 
like this <frontPage/> instead of <FrontPage/> but I don't think exporting it with small letter matters as long I'm exporting it as default, I'm allowed to change its name while importing.

Reply:
but if we agree as a team to write all component by starting with a capital letter, let's do that